### PR TITLE
Spanish documentation

### PR DESCRIPTION
**Pull request by Classy-Bear**
## Status
**IN DEVELOPMENT**

## Breaking Changes
NO

## Description
Spanish docs

## Progress
- [x] README
- [x] Cover page
- [x] Sidebar
- Introduction
  - [x] Getting Started
  - [ ] Why Bloc?
- Core Concepts
  - [ ] bloc
  - [ ] flutter_bloc
- [ ] Architecture
- [ ] Testing
- [ ] Naming Conventions
- Tutorials
  - Flutter
    - [ ] Counter
    - [ ] Timer
    - [ ] Infinite List
    - [ ] Login
    - [ ] Weather
    - [ ] Todos
    - [ ] Firebase Login
    - [ ] Firestore Todos
  - AngularDart
    - [ ] Counter
  - Flutter + AngularDart
    - [ ] Github Search
- Recipes
  - Flutter
    - [ ] Show SnackBar
    - [ ] Navigation
    - [ ] Bloc Access
- Tools
  - Extensions
    - [ ] IntelliJ
    - [ ] VSCode

### CODE REVIEW

File: docs/_navbar.md [hunk 1]
```diff
@@ -4,3 +4,4 @@
   - [:kr: ko-kr](/ko-kr/)
   - [:brazil: pt-br](/pt-br/)
   - [:czech_republic: cs](/cs/)
+  - [:es: spa](/spa/)
```
Comment (tenhobi) [inline docs/_navbar.md]: You should probably use ISO 639-1 code for Spanish, which is `es`. (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)

Comment (Classy-Bear) [reply docs/_navbar.md]: Oh ok, fixing it

Comment (tenhobi) [reply docs/_navbar.md]: I think a lot of the tools use these two letters, optionally you can also use another two letter for a country, like `en` (english) with `us` (USA) has. So something like `es-sp` or `es-mx`? See http://www.lingoes.net/en/translator/langcode.htm

File: docs/index.html [hunk 2]
```diff
@@ -28,7 +28,7 @@
         homepage:
           "https://raw.githubusercontent.com/felangel/bloc/master/README.md",
         loadSidebar: true,
-        coverpage: ["/", "/ko-kr/", "/fr/", "/pt-br/", "/cs/"],
+        coverpage: ["/", "/ko-kr/", "/fr/", "/pt-br/", "/cs/","/es/"],
```
Comment (tenhobi) [inline docs/index.html]: nit

```suggestion
        coverpage: ["/", "/ko-kr/", "/fr/", "/pt-br/", "/cs/", "/es/"],
```

File: docs/_navbar.md (hunk 1 above)
Comment (Classy-Bear) [reply docs/_navbar.md]: There is a problem with that, there no is official Spanish but we often use a standard but this standard does not belong to any country, should I put my country then? In my case is the Dominican Republic

File: docs/index.html (hunk 2 above)
Comment (Classy-Bear) [reply docs/index.html]: HAHA nice, that space got me XD

File: docs/_navbar.md (hunk 1 above)
Comment (tenhobi) [reply docs/_navbar.md]: I would wait for a suggestion from @felangel. Don't know what opinion he has on that. ;)

Comment (Classy-Bear) [reply docs/_navbar.md]: All right and thanks for the feedback 👍

Comment (felangel) [reply docs/_navbar.md]: I would recommend leaving it as es for now and if we need to add country codes later we can 👍

Comment (Classy-Bear) [reply docs/_navbar.md]: Sounds good to me

**Comment by jomazao** [comment]
Hi, now I have finished the  Introduction- Core concept and Architecture spanish translations how can I send it to you? This is my first time apporting to the comunity

**Comment by felangel** [comment]
> Hi, now I have finished the Introduction- Core concept and Architecture spanish translations how can I send it to you? This is my first time apporting to the comunity

Thoughts on merging this PR as is so that @Classy-Bear and @jomazao can split up the remaining work?

**Comment by Classy-Bear** [comment]
> > Hi, now I have finished the Introduction- Core concept and Architecture spanish translations how can I send it to you? This is my first time apporting to the comunity
> 
> Thoughts on merging this PR as is so that @Classy-Bear and @jomazao can split up the remaining work?

Sounds good to me, we will work on this documentation together.

**Pull request merged**